### PR TITLE
Building local storage for users

### PR DESCRIPTION
**Pull request by Caren-Koroeny**
- When the user changes the content of any input field, the data is saved to the local storage.
- When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @Caren-Koroeny,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that has no contribution from your pair programming partner. Kindly check the [Work and Submission Mode](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md#work-and-submission-mode) to learn more.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Caren-Koroeny** [comment]
- So sorry about this we are working on it asap.Thank you so much for that.

File: index.html [hunk 1]
```diff
@@ -181,14 +181,14 @@ <h2 class="title2 desktop">Contact me</h2>
             needs coding. I’d love to help with it
             </p>
             <div class="contact-form" >
```
Comment (huseyinbiyik) [inline index.html:183]: Great job team! But please make sure that your contact form is working on the desktop version. Contact form input areas are not clickable right now. Kindly check the gif below.
Screen Recording 2022-12-24 at 12 13 46 AM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @Caren-Koroeny and @shaaibu7 ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Good Readme file 👨‍💼
- Linter checks passes 🟢


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @Caren-Koroeny , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**